src/borg/helpers/parseformat.py

```diff
@@ -524,6 +546,15 @@ def _parse(self, text):
             self.proto = m.group("proto")
             self.path = os.path.normpath(m.group("path"))
             return True
+        m = self.s3_re.match(text)
+        if m:
+            self.proto = "s3"
+            self.user = m.group("profile") if m.group("profile") else m.group("access_key_id")
+            self._pass = True if m.group("access_key_secret") else False
```

Comment:
You set this here, but you do not use it anywhere.

Reply:
borgstore gets the original, repo url (with some processing). The parsing for Location is just so that we can detect that it's for s3 so we can pass the url off to borgstore.

References `Location.processed`: https://github.com/borgbackup/borg/blob/332ef28c7e3d73bf884206c98e6ca11b497acfa9/src/borg/repository.py#L110
`Location.processed` is the raw url with placeholders replaced: https://github.com/borgbackup/borg/blob/332ef28c7e3d73bf884206c98e6ca11b497acfa9/src/borg/helpers/parseformat.py#L501